packages/react-components/package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "@agoric/react-components",
-  "private": false,
```

Comment:
I can't remember why I added this, I think I was having trouble getting `react-components` to publish, but it might've been a matter of just adding:
```
"publishConfig": { "access": "public" }
```
below. I guess we'll find out, I assume `false` is the default here.

Reply:
yeah, it's the default. This was removed automatically by yarn.